website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -0,0 +1,357 @@
+| [`applicationConfig.APPSMITH_MAIL_USERNAME`](/getting-started/setup/instance-configuration/email/)               | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_PASSWORD`](/getting-started/setup/instance-configuration/email/)               | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_FROM`](/getting-started/setup/instance-configuration/email/)                   | `""`  |
+| [`applicationConfig.APPSMITH_REPLY_TO`](/getting-started/setup/instance-configuration/email/)                    | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_SMTP_AUTH`](/getting-started/setup/instance-configuration/email/)              | `""`  |
+| `applicationConfig.APPSMITH_MAIL_SMTP_TLS_ENABLED`       | `""`  |
+| [`applicationConfig.APPSMITH_DISABLE_TELEMETRY`](https://docs.appsmith.com/product/telemetry#disable-telemetry)           | `""`  |
+| `applicationConfig.APPSMITH_RECAPTCHA_SITE_KEY`          | `""`  |
+| `applicationConfig.APPSMITH_RECAPTCHA_SECRET_KEY`        | `""`  |
+| `applicationConfig.APPSMITH_RECAPTCHA_ENABLED`           | `""`  |
+| [`applicationConfig.APPSMITH_MONGODB_URI`](/getting-started/setup/instance-configuration/custom-mongodb-redis#custom-mongodb)                | `""`  |
+| [`applicationConfig.APPSMITH_REDIS_URL`](/getting-started/setup/instance-configuration/custom-mongodb-redis#custom-redis)                   | `""`  |
+| `applicationConfig.APPSMITH_ENCRYPTION_PASSWORD`         | `""`  |
+| `applicationConfig.APPSMITH_ENCRYPTION_SALT`             | `""`  |
+| [`applicationConfig.APPSMITH_CUSTOM_DOMAIN`](/getting-started/setup/instance-configuration/custom-domain/#dockerk8s)               | `""`  |
+
+Once you have made changes to the values file, run the below command.
+
+```bash
+helm install -f values.yaml stable-appsmith/appsmith --generate-name
+```
+
+## Publish Appsmith
+
+To make your Appsmith installation accessible on the internet, you need an Ingress controller set up. For more information on Ingress controllers, see the Configure Kubernetes Cluster section. If you haven't yet installed the Helm chart, you can run the following command to install the chart, enable the ingress controller, and publish Appsmith.
+
+```bash 
+helm install appsmith/appsmith --generate-name \
+  --set ingress.enabled=true \
+  --set ingress.annotations."kubernetes\.io/ingress\.class"=nginx \
+  --set service.type=ClusterIP
+```
+
+If you have installed Appsmith Helm chart, run the `helm upgrade` command to upgrade the existing installation.
+
+    ```
+    helm upgrade --set ingress.enabled=true appsmith appsmith/appsmith
+
+    # Or this command if you are using values.yaml file
+    helm upgrade --values values.yaml appsmith appsmith/appsmith
+    ```
+
+## Update Appsmith
+
+Appsmith installation can be updated manually or by enabling auto update.
+
+### Auto update
+
+The default Appsmith helm installation disables the auto update (_Recommended_). You have an option to enable auto update by either setting it in values file or using a command.
```

Comment:
The `recommended` tag is confusing here. Are we recommending _the auto-update feature itself_, or are we recommending _keeping auto-update disabled_?

```suggestion
The default Appsmith helm installation disables auto update (_recommended_). You have an option to enable auto update by either setting it in values file or using a command.
```

Reply:
As per existing doc, I kept it as is.